packages/theme_tailor/lib/src/template/theme_class_template.dart

```diff
@@ -22,8 +22,20 @@ class ThemeClassTemplate {
     final constructorBuffer = StringBuffer();
     final fieldsBuffer = StringBuffer();
 
-    config.fields.forEach((key, value) {
-      constructorBuffer.write('required this.$key,');
+    final sortedFields = Map.fromEntries(config.fields.entries.sorted((a, b) {
+      if (a.value.isNullable && !b.value.isNullable) {
+        return 1;
+      } else if (!a.value.isNullable && b.value.isNullable) {
+        return -1;
+      }
+      return 0;
```

Comment:
This should be in the `Field` class, also we should also sort by name here if we want to take care of it. I think it's best for the `Field` to extend/mix with `Comparable`